src/pages/Company/Company.js

```diff
@@ -1,7 +1,50 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import styled from 'styled-components';
+import CompanyHead from './components/CompanyHead';
+import CompanyInfo from './components/CompanyInfo';
+import CompanyAside from './components/CompanyAside';
 
 function Company() {
-  return <div>Company</div>;
+  const [companyData, setCompanyData] = useState([]);
+
+  useEffect(() => {
+    fetch('/data/COMPANY_DATA.json')
+      .then(res => res.json())
+      .then(data => setCompanyData(data));
```

Comment:
```suggestion
      .then(([data]) => setCompanyData(data));
```
0번째 아이템만 필요한거라면 위와 같이 표현해도 될 것 같네요!

Reply:
리뷰 감사합니다! 수정해서 다시 푸쉬 할 수 있도록 하겠습니다!